core/platform/users/gae_current_user_services.py

```diff
@@ -74,9 +74,13 @@ class _FakeUser(ndb.Model):
         return None
 
 
-def get_user_id(user):
-    """ Given an user object, get the user id. """
-    return user.user_id()
+def get_current_user_id():
+    """ Gives the user_id of current user."""
```

Comment:
Nits: drop space after first """; Gives --> Gets (and indicate that None is returned if there is no current user).